forge/build.gradle

```diff
@@ -0,0 +1,132 @@
+/*
+ * This file is part of Baritone.
+ *
+ * Baritone is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Baritone is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Baritone.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import baritone.gradle.task.CreateDistTask
+import baritone.gradle.task.ProguardTask
+
+plugins {
+    id "com.github.johnrengelman.shadow" version "7.0.0"
+}
+
+archivesBaseName = archivesBaseName + "-forge"
+
+minecraft {
+    forge {
+        mixinConfig = ["mixins.baritone.json"]
+    }
+
+    launches.client = {
+        args.addAll(["--tweakClass", "org.spongepowered.asm.launch.MixinTweaker"])
+        for (int i = 0; i < args.size(); i++) {
+            if (args[i] == "-mixin.config") {
+                args[i] = "--mixin"
+            }
+        }
```

Comment:
An explanatory comment for why we do this would be nice.

Reply:
```suggestion
        // old mixin (0.7.11) uses a different flag for specifying configs
        // in the launch args
        for (int i = 0; i < args.size(); i++) {
            if (args[i] == "-mixin.config") {
                args[i] = "--mixin"
            }
        }
```